### DISCUSSION TITLE
LGPL. Question about the license

### DISCUSSION THREAD

**Discussion by ayrat555** [brndnmtthws/dryoc#34] [Q&A] [answered]
Hey, @brndnmtthws 

Can I use this project to create a library for another programming language. should my library under the same license? can I use MIT?

**Comment by brndnmtthws** [comment, marked as answer]
That sounds fine by me, but I don't really know the legal details. I think if you're merely including the library, and not the source code directly, it should be fine to do it under MIT. With LGPL it only matters if you make changes to the original library, in which case you'd be required by the license to share those changes.

It's also something that's nearly impossible to detect or enforce, and I don't have any interest in getting involved in any copyright/legal battles so practically speaking you can likely do whatever you want.

I'd prefer if people respected the license and shared any improvements they make (if they do that), but I'm not going to sue anyone over it.
